.github/ISSUE_TEMPLATE/create-new-doctype-template.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Create new doctype template
+about: For issues where a templateer needs to create a new doctype template.
+title: Create {doctype} template
+labels: templates
+assignees: ''
+
+---
+
+# Create {doctype} template
+
+## Description
+### Doctype Name: {doctype}
+### Directory: https://github.com/thegooddocsproject/templates/{directory_name}
+### Sub-tasks
```

Comment:
I'm using the Conventional Comments method to respond to your PR.

**Question:** Could you explain the "Doctype Name" and "Directory" fields to me so that I can understand them? As long as they make sense to you when you create the issue and would make sense to someone reading the issue when they reading it, these fields could be helpful. However, their purpose isn't immediately clear to me.

Reply:
So each of these issues is going to create template files for one specific "doc type"; that's a convention coming from Cameron's writing about the base template. I had some reservations about us coining this term since it could be confused with the "doctype" used in development when writing HTML, but I haven't come up with anything better. An example of a doc type in our system would be "API Reference" or "Tutorial". I debated whether this field was needed (might be obvious from the title of the issue). "Directory" is there so a newbie picking up one of these issue can go directly to where they need to be to start creating files, for example "https://github.com/thegooddocsproject/templates/tutorial". It's not that obvious I think for someone new to the project who isn't that familiar with our repo structure. And when we get to issues for nested doc types (like API Reference under Reference), the directory the files need to be created in won't be visible from the top-level of the templates repo.

Reply:
nitpick (non-blocking): We should be consistent is our spelling of "doctype/doc type/doc_type/doc-type/Doc Type/..." I've been using doctype, but only because I haven't got a better idea. I'm happy to switch if someone can suggest something better. My mild preference order would be:
1. doctype
2. doc-type

Reply:
Yeah the reason I've started to use "doc type" is to avoid confusion with the [doctype](https://www.w3schools.com/tags/tag_doctype.asp) declaration in HTML markup. Of the two that you recommended for preference then, my vote would be for "doc-type".